src/main/java/duke/Event.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    private LocalDate startTime;
+    private LocalDate endTime;
+
+    public Event(String s, String startTime, String endTime) {
```

Comment:
Perhaps can use a more meaningful name for "s"
```suggestion
    public Event(String s, String startTime, String endTime) {
```